tests/TestCase/Validation/ValidationTest.php

```diff
@@ -2879,6 +2879,18 @@ public function testIsArray()
     }
 
     /**
+     * Test isScalar
+     *
+     * @return void
+     */
+    public function testIsScalar()
+    {
+        $this->assertTrue(Validation::isScalar(1));
+        $this->assertTrue(Validation::isScalar(''));
```

Comment:
but after the discussion below: `$this->assertFalse(Validation::isScalar(null));`... maybe ;)